Makefile

```diff
@@ -3,3 +3,18 @@ lint.py:
 
 requirements-local.py:
 	pip install -q -r requirements/local.txt --exists-action w
+
+tools-dev:
+	SERVER_APP=tools-dev docker compose --env-file config/app.config up --build
```

Comment:
Use [BuildKit and define `target](https://docs.docker.com/build/building/multi-stage/#differences-between-legacy-builder-and-buildkit)` so that compose only build those stages required by a `target`.